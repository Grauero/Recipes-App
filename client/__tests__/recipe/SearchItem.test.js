import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import SearchItem from '../../src/components/recipe/SearchItem';

it('matches snapshot', () => {
  const props = { _id: 'id', name: 'name', likes: 0 };
  const component = shallow(<SearchItem {...props} />);

  expect(toJSON(component)).toMatchSnapshot();
});
