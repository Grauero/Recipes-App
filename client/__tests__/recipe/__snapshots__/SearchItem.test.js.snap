// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<li>
  <Link
    replace={false}
    to="/recipes/id"
  >
    <h4>
      name
    </h4>
  </Link>
  <p>
    Likes: 
    0
  </p>
</li>
`;
